pkg/event/target/redis.go

```diff
@@ -185,7 +185,10 @@ func (target *RedisTarget) send(eventData event.Event) error {
 	}
 
 	if target.args.Format == event.AccessFormat {
-		data, err := json.Marshal([]interface{}{eventData.EventTime, []event.Event{eventData}})
+		data, err := json.Marshal(struct {
```

Comment:
Okay so the change itself looks good but to keep backward compatibility - It is perhaps better to just do 

```suggestion
		data, err := json.Marshal([]struct {
```

It allows existing parsers to not fail, you can also try to define this as 

```
type RedisAccessEvent struct {
             Event     []event.Event
	     EventTime string
}
```
```suggestion
		data, err := json.Marshal([]RedisAccessEvent{
```
```

Reply:
That makes sense more sense, thanks. I've tested that locally and pushed an updated PR with that change.